action.yml

```diff
@@ -34,6 +34,10 @@ inputs:
       environment with the assumed role credentials rather than with the provided
       credentials
     required: false
+  web-identity-token-file:
+      description: >-
+        Read the web identity token file from the provided file system path in order to 
+        assume an IAM role using a web identity on an EKS worker node
```

Comment:
Does this by default considers web-identity-token-file as required **false**?

Reply:
Good catch @paragbhingre .  Updated.